kornia/augmentation/_2d/geometric/base.py

```diff
@@ -208,7 +208,7 @@ def inverse_boxes(
 
         return output
 
-    def inverse_keypoints(
+    def inverse_keypoints(  # noqa: D417
```

Comment:
why needed here ? there's documentation

Reply:
D417 checks if all arguments are documented. The kwargs aren't documented. (Verified adding a kwargs entry clears it)